cg-spec/editors_draft.html

```diff
@@ -2542,6 +2557,16 @@ <h2>RDF-star Semantics</h2>
       and <dfn data-cite="RDF11-MT#dfn-equivalent" data-lt="equivalent">equivalence</dfn>.
     </p>
 
+    <section id="motivation-and-caveats">
+      <h2>Motivation and Caveats</h2>
+
+      <p>The RDF-star semantics makes a very unusual design decision: embedded triples are referentially opaque. Therefore IRIs in embedded triples do denote their syntactic representation but not their interpretation like they do everywhere else in RDF. The purpose of this design is to represent statements with the utmost precision, almost as if they were literals (except blank nodes which co-denoten as usual). This supports application-centric use cases like Explainable AI or <a href="https://w3c.github.io/rdf-star/UCR/rdf-star-ucr.html#annotate-commit-deltas">Commit History</a> that would otherwise be much harder or impossible to implement in RDF.</p>
+
+      <p>The downside of this design is that IRIs in embedded triples don't co-denote like they do in the statement that the embedded triple represents. This can have unexpected effects and impairs the ability of embedded triples to reliably model common constructs like attributed relations in property graph style, reification or n-ary relations. It is common on the semantic web that data integration processes replace IRIs by co-denoting terms from other vocabularies, and this ability is indeed essential to the conception of the semantic web as a shared knowledge space without centralized vocabularies. Statements so modified still have the same meaning as only co-denoting terms are exchanged. Embedded triples however, although in most use cases supposed to present the hinge between a statement and its annotations, won't follow suit as they are fixed to their syntactic form. Annotations therefore may loose their connection with the statement they were meant to annotate.</p>
```

Comment:
```suggestion
      <p>One potential downside of this design is that IRIs in quoted, i.e., opaquely embedded, triples don't co-denote like they would in the triple that the quoted triple represents. This can have unexpected effects with incautious use, which can also impair users' ability to use quoted triples to model common constructs like attributed relations in property-graph&ndash;style, reification, or n-ary relations. Users must take care to only use asserted, i.e., transparently embedded, triples for such purposes.</p>
      <p>Another usage pattern common on the semantic web is data integration processes, whereby IRIs in a graph may be replaced by co-denoting terms from other vocabularies. This ability is indeed essential to the conception of the semantic web as a shared knowledge space without centralized vocabularies. Statements so modified still have the same meaning as only co-denoting terms are exchanged. While asserted (transparently embedded) triples in many use cases serve as the hinge between a statement and its annotations, quoted (opaquely embedded) triples don't follow suit, as they are fixed to their syntactic form. With incautious use of quoted triples, which are not touched by such co-denoting substitutions as described above, annotations therefore may lose their connection with the statement they were meant to annotate, which <em>are</em> touched by co-denoting substitutions.</p>
```

Reply:
I like the start of your changes. Then IMO you've cut too much. In the end, "Users must take care to only use asserted, i.e., transparently embedded, triples for such purposes." is wrong, as that won't help. But if my text was better aligned with [example 35](https://pr-preview.s3.amazonaws.com/w3c/rdf-star/185/524a524...rat10:5f77b06.html#ref-opacity-owl-entailment-2) that would be easier to understand (and probably could be shortened quite a bit).